projects/concurrency/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func (c *Cache) GetKeys() []int {
+	keys := make([]int, 0, len(c.storage))
+	for key := range c.storage {
+		keys = append(keys, key)
+	}
+	return keys
+}
+func TestPut(t *testing.T) {
+	cache := NewCache(2)
+	t.Run("Case1/Below capacity", func(t *testing.T) {
+		cache.Put(1, "one")
+		cache.Put(2, "two")
+		actual := cache.GetKeys()
+		require.Equal(t, []int{1, 2}, actual)
```

Comment:
The iteration order of maps in Go is unspecified, so sometimes this will actually be `[]int{2, 1}` instead (try running `go test ./... -count=10` which will run the tests 10 times, and you'll see that sometimes they pass and sometimes they fail).

Can you think how you could achieve the same aim here _without_ relying on order?

Reply:
I used this pattern : 
```go
	_, exists := cache.storage[1]
		require.True(t, exists)
```